src/hostname/hostnamectl.c

```diff
@@ -448,11 +448,11 @@ static int show_all_names(sd_bus *bus) {
                             &hardware_serial_reply,
                             NULL);
         if (r < 0)
-                log_full_errno(sd_bus_error_has_names(
+                log_full_errno((sd_bus_error_has_names(
                                                &error,
                                                BUS_ERROR_NO_HARDWARE_SERIAL,
                                                SD_BUS_ERROR_INTERACTIVE_AUTHORIZATION_REQUIRED,
-                                               SD_BUS_ERROR_UNKNOWN_METHOD) ? LOG_DEBUG : LOG_WARNING,
+                                               SD_BUS_ERROR_UNKNOWN_METHOD) || r == -ENOENT) ? LOG_DEBUG : LOG_WARNING,
```

Comment:
Also, here. Maybe the issue is in the service side?

Reply:
Haven't got a scooby. I don't see anything obvious, but then again I didn't implement the feature, just trying to get the CI back to green. Let's do that first please, then @poettering can look in more details and change this if there's a better solution